src/app/src/theme.ts

```diff
@@ -36,11 +36,18 @@ const theme = extendTheme({
     },
     colors: {
         tooltip: {
-            500: '#465EB5',
+            500: '#45B565',
         },
         progress: {
             500: '#000000',
         },
+        green: {
+            50: '#F1F8F3',
+            100: '#DEECE2',
+            300: '#ADCDB7',
+            600: '#3DA45A',
+            900: '#235F34',
+        },
```

Comment:
Interesting! I'm only parroting what I'm used to from azavea/iow-boundary-tool. I can imagine with six or seven shades that 'medium' becomes less helpful (extra medium?) I'm wondering if you contemplated that on any other projects?

Chakra also guides toward numbers: https://chakra-ui.com/docs/styled-system/theme

Reply:
I thought we made that choice in a recent Civic Apps project, but can't find the example I was looking for, but also for that we had 5 distinct colors with a light and dark variant, so that may not apply here since this is a monochromatic gradient.